am-lib/src/test/java/integration/uk/gov/hmcts/reform/amlib/GetRolePermissionsIntegrationTest.java

```diff
@@ -106,6 +110,47 @@ void whenInsufficientRolePermissionsShouldReturnNull() {
         assertThat(rolePermissions).isNull();
     }
 
+    @Test
+    void whenRolePermissionIsHighestThenShowAllAttribute() {
```

Comment:
@abhijitdiwan on the basis that the tests does not exercise any new execution path it adds no much more value. 

However as see value in it as it shows that not only exact match on security classification is possible but all security classifications at the same or below level of hierarchy are fetched. On the above basis I suggest changing existing `whenGettingRolePermissionsShouldReturnPermissionsAndSecurityClassificationsForSpecifiedRole` by replicating the scenario that you proposed.

In other words no new test please just change to existing one.

Reply:
Updated in existing method. Please have a look